clustered/server/src/test/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntityTest.java

```diff
@@ -106,7 +107,7 @@ public void testCreateSharedServerStore() throws Exception {
 
     assertThat(defaultRegistry.getStoreManagerService().getStores(), containsInAnyOrder(defaultStoreName));
     assertThat(defaultRegistry.getStoreManagerService()
-        .getSharedResourcePoolIds(), containsInAnyOrder(defaultSharedPool));
```

Comment:
see no difference other than a blank.

Reply:
Must be accidental formatting - will revert.